codebuild/bin/s2n_setup_env.sh

```diff
@@ -134,10 +134,35 @@ rm -rf libcrypto-root && ln -s "$LIBCRYPTO_ROOT" libcrypto-root
 # Set the libfuzzer to use for fuzz tests
 export LIBFUZZER_ROOT=$LIBFUZZER_INSTALL_DIR
 
+#check if the path contains test dep X, if not and X exists, add to path
+path_overrides="$PYTHON_INSTALL_DIR/bin
+$OPENSSL_1_1_1_INSTALL_DIR/bin
+$GNUTLS_INSTALL_DIR/bin
+$SAW_INSTALL_DIR/bin
+$Z3_INSTALL_DIR/bin
+$SCAN_BUILD_INSTALL_DIR/bin
+$PRLIMIT_INSTALL_DIR/bin
+$LATEST_CLANG_INSTALL_DIR/bin
+`pwd`/codebuild/bin
+~/.local/bin"
+
+testdeps_path(){
+    echo -ne "checking $1 is in the path..."
+    if [[ ! "$PATH" =~ "$1" ]]; then
+        if [[ -d "$1" ]]; then
+            export PATH="$1:$PATH"
+            echo -e "added"
+        else
+            echo -e "doesn't exist"
+        fi
+    else
+        echo -e "already in path"
+    fi
+}
+
+for i in $path_overrides; do testdeps_path "$i" ;done
+
 # Just recording in the output for debugging.
-if [ -f "/etc/lsb-release" ]; then
-  cat /etc/lsb-release
-fi
```

Comment:
Why is this no longer required?

Reply:
Its duplicated elsewhere (printenv), but reverted for now.